internal/characters/kokomi/burst.go

```diff
@@ -124,17 +125,28 @@ func (c *char) burstActiveHook() {
 			return false
 		}
 
-		c.Core.Player.Heal(player.HealInfo{
-			Caller:  c.Index,
-			Target:  -1,
-			Message: "Ceremonial Garment",
-			Src:     burstHealPct[c.TalentLvlBurst()]*c.MaxHP() + burstHealFlat[c.TalentLvlBurst()],
-			Bonus:   c.Stat(attributes.Heal),
-		})
-
-		if c.Base.Cons >= 2 {
-			c.c2()
+		heal := burstHealPct[c.TalentLvlBurst()]*c.MaxHP() + burstHealFlat[c.TalentLvlBurst()]
+		for _, char := range c.Core.Player.Chars() {
+			src := heal
+
+			// C2 handling - believe this is an additional instance of flat healing
```

Comment:
change the comment to just // C2 handling in both places, because it's not an additional instance of flat healing